<launch>
  <!-- Load the URDF int the ROS Parameter Server -->
  <param name="robot_description" command="$(find xacro)/xacro '$(find gazebo_demo)/robots/4limb_robot.urdf.xacro'"/>
  <!-- other urdf may be used -->

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

  <!-- gait -->
  <node name="gait_publisher" pkg="gazebo_demo" type="gait.py" respawn="true"/>
  <node name="gait_ik" pkg="gazebo_demo" type="gait_ik" respawn="true" output="screen"/>

  <!-- rviz -->
  <node name="$(anon rviz)" pkg="rviz" type="rviz" args="-d $(find gazebo_demo)/launch/gait_check.rviz"/>

</launch>
